packages/bundle-size/README.md

```diff
@@ -24,6 +24,50 @@ export default {
 };
 ```
 
+### Configuration
+
+For custom advanced behavior of `bundle-size`, you can create a `bundle-size.config.js` in the root of your project directory (next to `package.json`).
+
+```
+my-proj/
+├─ src/
+├─ bundle-size.config.js
+├─ node_modules/
+├─ bundle-size/
+│  ├─ Fixture.fixture.js
+├─ package.json
+```
+
+A global configuration can also be used for monorepo scenarios
+
+```
+my-proj-a/
+├─ src/
+├─ node_modules/
+├─ bundle-size/
+│  ├─ Fixture.fixture.js
+├─ package.json
+my-proj-b/
+├─ src/
+├─ node_modules/
+├─ bundle-size/
+│  ├─ Fixture.fixture.js
+├─ package.json
+bundle-size.config.js
+
+```
+
```

Comment:
I'd add explicit type annotation for config API, so its obvious what is supported

```suggestion
#### Config API

```ts
type WebpackConfig = import("webpack").Configuration

interface Config {
  webpack: (config: WebpackConfig) => WebpackConfig
}
```

```